### PR TITLE
Refactoring.calibration.pixel2world

### PR DESCRIPTION
**Pull request by bjornborg**
Note I also added some modification to the make command buildIncremental for compiling the tools as well.

### CODE REVIEW

File: code/tool/camera-projection/src/cameraprojection.cpp [hunk 1]
```diff
@@ -120,8 +122,10 @@ void CameraProjection::setUp()
 {
   odcore::base::KeyValueConfiguration kv = getKeyValueConfiguration();
 
-  m_cameraName = kv.getValue<std::string>("core-tool-camera-projection.cameraname");
-  m_transformationMatrixFileName = m_cameraName + "-pixel2word-matrix.csv";
+  m_cameraName = 
+      kv.getValue<std::string>("core-tool-camera-projection.cameraname");
+  m_debug = (kv.getValue<int32_t>("core-tool-camera-projection.debug") == 0);
+  m_transformationMatrixFileName = "/opt/opendlv.core.configuration/" + m_cameraName + "-pixel2word-matrix.csv";
```
Comment (chrberger) [inline code/tool/camera-projection/src/cameraprojection.cpp]: You should use the relative path of this module (i.e., "./") and make sure that the corresponding docker-compose.yml file contains the right setting for working directory, i.e. -w

File: usecases/latest/calibration.projection/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    odsupercomponent:
+        build: .
+        # image: seresearch/opendavinci-ubuntu-16.04-complete:latest
+        network_mode: host
+        volumes:
+            - .:/opt/opendlv.core.configuration
+        command: "/opt/od4/bin/odsupercomponent --cid=${CID} --verbose=1 --configuration=/opt/opendlv.core.configuration/configuration"
+    
+    # Micro-service for projection calibration
+    tool-camera-projection:
```
Comment (chrberger) [inline usecases/latest/calibration.projection/docker-compose.yml:33]: Here, the working directory path is missing (cf. above)

File: usecases/latest/test-camerareplay/configuration [hunk 3]
```diff
@@ -78,7 +78,7 @@ odsupercomponent.pulsetimeack.exclude = odcockpit
 #
 core-tool-camera-replay.debug = 1       # 1 = show recording (requires X11), 0 = otherwise.
 core-tool-camera-replay.sourcename = AxisCamera0
-core-tool-camera-replay.filename = highway.avi
+core-tool-camera-replay.filepath = /opt/Videos/highway.avi
```
Comment (chrberger) [inline usecases/latest/test-camerareplay/configuration]: We should use relative paths preferably.

**Review by chrberger — CHANGES_REQUESTED** [review]

**Comment by bjornborg** [comment]
Reqested changes are now done.

**Comment by chrberger** [comment]
Thanks, looks fine

**Review by chrberger — APPROVED** [review]
@olbender ?

**Pull request merged**